systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/RollingUpdateST.java

```diff
@@ -245,8 +245,7 @@ void testRecoveryDuringKafkaRollingUpdate(ExtensionContext extensionContext) {
     @ParallelNamespaceTest
     @Tag(ACCEPTANCE)
     @Tag(COMPONENT_SCALING)
-    @KRaftNotSupported("Zookeeper is not supported by KRaft mode and is used in this test case")
-    void testKafkaAndZookeeperScaleUpScaleDown(ExtensionContext extensionContext) {
+    void testKafkaScaleUpScaleDown(ExtensionContext extensionContext) {
```

Comment:
ZooKeeper is still there - it is only skipped when running in the KRaft mode. So maybe its not needed to rename the test?

Reply:
updated. Thanks.